selenium/src/test/java/eventkitui/test/AuthenticationTest.java

```diff
@@ -0,0 +1,89 @@
+package eventkitui.test;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import eventkitui.test.page.*;
+import eventkitui.test.util.Utils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.rules.TestName;
+import org.openqa.selenium.WebDriver;
+
+
+/**
+ * Tests Login and Logout functionality
+ */
+public class AuthenticationTest {
```

Comment:
This test originally tested the gx login page, I'm now using the admin backend to allow local testing. I'd like to remove this test but I will keep it in here if we want to test users against geoaxis